networkx/readwrite/adjlist.py

```diff
@@ -60,6 +60,14 @@ def generate_adjlist(G, delimiter=" "):
     --------
     write_adjlist, read_adjlist
 
+    Notes
+    -----
+    If spaces are embedded in the names of nodes, the parser in read_adjlist will
+    be confused because space is the default delimiter. The parser will not
+    know if a space belongs in the node name or is just a delimiter.
+    To avoid this problem, specify an alternate delimiter when spaces are
+    valid in node names.
```

Comment:
```suggestion
    The default `delimiter=" "` will result in unexpected results if node names contain
    whitespace characters.
```

Just a minor wording suggestion - I think keeping this as short as possible would be best. Note also that the advice to specify an alternate delimiter may not be possible if the data was generated by someone other than the user.

Reply:
Okay. I'll add that and correct it. Thank you @rossbar